Instastories.py

```diff
@@ -18,32 +18,43 @@
 
 
 ################# UTILS FUNCTIONS #########################
-def get_session_id(username, password):
+
+def save_settings(settings):
+    with open("settings.json", "w+") as settings_json:
```

Comment:
save this at the top of the file as SETTINGS_PATH = "settings.json"

Replace all usages